app/views/layouts/base.html.erb

```diff
@@ -68,7 +68,7 @@ See COPYRIGHT and LICENSE files for more details.
 <% initial_classes = initial_menu_classes(side_displayed, show_decoration) %>
 <div id="wrapper" style="<%= initial_menu_styles(side_displayed) %>" class="<%= initial_classes %>">
   <% if show_decoration %>
-    <header class="op-app-header">
+    <header class="op-app-header<%= ' op-app-header--development' if Rails.env.development? %>">
```

Comment:
Just a minor thing, but please use an underscore instead of the double dash ( so `op-app-header_development`), as "development" is rather a modifier than an element.

Reply:
I'm confused, I thought that using double dashes is part of BEM convention, but now found a link in documentation to bem.info and page on naming [have multiple naming schemas](https://en.bem.info/methodology/naming-convention/#alternative-naming-schemes), do we use the ["default" one](https://en.bem.info/methodology/naming-convention/#naming-rules)?

Reply:
Unfortunately (due to historic reasons) we have some mixed up notation for class names in the core. Basically we currently use `block--element_modifier`. However, it is not consistently used throughout the application, especially the old code blocks still have another notiation (which is also not BEM 🙈 ). 
Inside our Primer repo, (and all components coming from it) we stick however to BEM.